azure-pipelines.release-vnext.yml

```diff
@@ -0,0 +1,82 @@
+pr: none
+trigger: none
+
+# Customize build number to include major version
+# Example: v9_20201022.1
+name: 'v9_$(Date:yyyyMMdd)$(Rev:.r)'
+
+variables:
+  - group: 'Github and NPM secrets'
+  - template: .devops/templates/variables.yml
+    parameters:
+      skipComponentGovernanceDetection: false
+  - name: release.vnext # Used to scope beachball to release only vnext packages
+    value: true
+
+pool: 'Self Host Ubuntu'
+
+# schedules:
+#   # minute 0, hour 7 in UTC (11pm in UTC-8), Every monday
+#   # https://docs.microsoft.com/en-us/azure/devops/pipelines/build/triggers?tabs=yaml&view=azure-devops#supported-cron-syntax
+#   - cron: '0 7 * * 1'
+#     # will be 12am during daylight savings time unless trigger is updated
+#     displayName: 'Scheduled release (Every monday)'
+#     branches:
+#       include:
+#         - master
+
+workspace:
+  clean: all
+
+steps:
+  - template: .devops/templates/tools.yml
+
+  - script: |
+      git config user.name "Fluent UI Build"
+      git config user.email "fluentui-internal@service.microsoft.com"
+      git remote set-url origin https://$(githubUser):$(githubPAT)@github.com/microsoft/fluentui.git
+    displayName: Authenticate git for pushes
+
+  - task: Bash@3
+    inputs:
+      filePath: yarn-ci.sh
+    displayName: yarn
+
+  - script: |
+      yarn run:published build --production --no-cache
+    displayName: yarn build
+
+  - script: |
+      yarn run:published test
+    displayName: yarn test
+
+  - script: |
+      yarn run:published lint
+    displayName: yarn lint
+
+  - script: |
+      yarn beachball publish -b origin/master -n $(npmToken) --access public -y
+      git reset --hard origin/master
+    env:
+      GITHUB_PAT: $(githubPAT)
+    displayName: Publish changes and bump versions
+
+  # Run this near the end because it's more likely to fail than the artifact upload tasks, and its
+  # failure doesn't need to block anything else
```

Comment:
Were you intentionally keeping this disabled until the release notes script is updated? And regardless you can remove this comment.
```suggestion
```

Reply:
yes the plan is the update the release notes script, that would be very low priority right now. I've kept the step commented and modified to comment to reflect that